level-1/내적.py

```diff
@@ -0,0 +1,5 @@
+def solution(a, b):
+    answer = 0
+    for i in range(len(a)):
+        answer += a[i] * b[i]
```

Comment:
Try to convert this code using Python's list conversion. It should be a good exercise.

Reply:
Completed fixing code with list comprehension.